cdap-docs/admin-manual/source/operations/cdap-ui.rst

```diff
@@ -38,5 +38,234 @@ and are confident of their suitability and compatibility. Earlier versions (and
 suppliers) of components may work, but have not necessarily been either tested or
 confirmed compatible.
 
+UI Customization
+---------------------------------
+Certain properties in the UI can be customized. The ``ui/server/config/themes/default.json`` file contains these properties, along with their default values. As of version 1.0, here is the list of properties that can be customized, and their expected values:
+
+.. list-table::
+   :widths: 15 30 20 30
+   :header-rows: 1
+
+   * - Property Name
+     - Property Attributes
+     - Description
+     - Example
+
+   * - ``brand-primary-color``
+     - No attributes
+     - The brand color to be used throughout UI. Can be in hex/rgb/rgba format.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "brand-primary-color": "#3b78e7"
+
+
+   * - ``navbar-color``
+     - No attributes
+     - The background color of the top nav bar. Can be in hex/rgb/rgba format.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "navbar-color": "rgba(51, 51, 51, 1)"
+
+   * - ``font-family``
+     - No attributes
+     - Comma-separated list of font families to be used in UI, in decreasing order of priority.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "font-family": "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif"
+
+   * - ``product-name``
+     - No attributes
+     - The product name to be used in UI. This will displayed on page titles, 'Welcome' modal, and the 'About' modal.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "product-name": "CDAP"
+
+   * - ``product-description``
+     - No attributes
+     - The product description to be used in 'Welcome' modal.
+     - .. container:: highlight
+
+        .. parsed-literal::
+
+          "product-description": "CDAP is an open source framework that simplifies data application development, data integration, and data management."
+
+   * - ``product-logo-navbar``
```

Comment:
there is a section at https://docs.cdap.io/cdap/5.1.0-SNAPSHOT/en/developer-manual/pipelines/developing-plugins/presentation-plugins.html#H3359 that describes this in much greater detail. Should we refer to it somehow instead of repeating the text?

Reply:
What do you mean by somehow refer to the section which is not connected to how you customize CDAP. Not sure, how these two are related. I like this to a separate section focused on user experience customization. We will be adding more stuff to this in coming releases.

Reply:
I think that Bhooshan means here is that the expected JSON formatting of the logos here follows the same pattern as the plugin icon described in the link, so we can refer to that section instead of repeating the text here.